### PR TITLE
Sign TX Preview V2

### PR DESCRIPTION
**Pull request by aristidesstaffieri**
Refactors transaction signing to our new designs: https://www.figma.com/file/C3G0a4Gd6RQyplRBppGDsL/Freighter-(SDS-v3)?type=design&node-id=228-16058&mode=design&t=qghcnSWEqWuUztWF-0
Adds Summary, Details, and Data tabs with different granularity of data in each.
Refactors the invocation tree renderer into human readable invocation KeyValue type of renders.
Refactors warnings to fit inline pre new preview design.
Screenshot 2024-02-13 at 1 43 30 PM

Screenshot 2024-02-13 at 1 43 45 PM
Screenshot 2024-02-13 at 1 44 00 PM
Screenshot 2024-02-13 at 1 44 13 PM

### CODE REVIEW

File: extension/src/popup/views/SignTransaction/Preview/Data/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import "./styles.scss";
+import { CopyText, Icon } from "@stellar/design-system";
```
Comment (piyalbasu) [inline extension/src/popup/views/SignTransaction/Preview/Data/index.tsx:4]: NIT: `import { CopyText, Icon } from "@stellar/design-system";` should come before the style import. This should be throwing an eslint warning

File: extension/src/popup/views/SignTransaction/Preview/Summary/index.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import "./styles.scss";
+import { Icon } from "@stellar/design-system";
```
Comment (piyalbasu) [inline extension/src/popup/views/SignTransaction/Preview/Summary/index.tsx:4]: same issue as above here

**Review by piyalbasu — APPROVED** [review]

File: extension/src/popup/views/SignTransaction/Preview/Data/index.tsx (hunk 1 above)
Comment (aristidesstaffieri) [reply extension/src/popup/views/SignTransaction/Preview/Data/index.tsx]: weird I need to figure out why that rule doesn't trigger for me. I get other lint warnings, and even other import order lint warnings but those didn't trigger for some reason.
Fixed in efda2d157b9925c274d733854f10634fea9e495d

File: extension/src/popup/views/SignTransaction/Preview/Summary/index.tsx (hunk 2 above)
Comment (aristidesstaffieri) [reply extension/src/popup/views/SignTransaction/Preview/Summary/index.tsx:3]: fixed in efda2d157b9925c274d733854f10634fea9e495d

**Pull request merged**